sql_bench/README.md

```diff
@@ -0,0 +1,31 @@
+# Local e2e SQL Benchmarks
```

Comment:
nit: I think we should avoid putting it on the top-level directory, we have already way too many, e.g., `scripts`, `standalone`, `develop`, ... 

(The cleanest I've seen is https://github.com/tigerbeetle/tigerbeetle )

Maybe we can put this in `scripts` or `develop` (Maybe many can be put under `develop`

Reply:
good suggestion, I think under `develop` makes sense.